app/models/queries/named_policy_queries.rb

```diff
@@ -24,9 +24,11 @@ def self.can_be_skipped?(hbx_id)
     end
 
     def self.shop_quiet_period_enrollments(effective_on, enrollment_statuses)
-      feins = Organization.where(:"employer_profile.plan_years" => {:$elemMatch => {
-        :start_on => effective_on, :aasm_state => 'enrolled'
-        }}).pluck(:fein)
+
+      feins =  BenefitSponsors::BenefitSponsorships::BenefitSponsorship.where(:benefit_applications => {
```

Comment:
Quiet period query needed only for initial employers....update query with predecessor_id check to make sure its pulling only initial employers